vspace/src/main/java/edu/asu/diging/vspace/web/publicview/ExhibitionSpaceController.java

```diff
@@ -49,12 +51,27 @@
     @Autowired
     private SequenceHistory sequenceHistory;
 
-    @RequestMapping(value = "/exhibit/space/{id}")
-    public String space(@PathVariable("id") String id, Model model) {
+    @RequestMapping(value = { "/exhibit/space/{id}", "/preview/{previewId}/space/{id}", "/preview/{previewId}" })
+    public String space(@PathVariable(name = "id", required = false) String id,
+            @PathVariable(name = IPreviewConstant.PREVIEW_ID, required = false) String previewId, Model model) {
+
+        if (previewId != null) {
+            Exhibition exhibition = (Exhibition) exhibitManager.getStartExhibition();
+            ExhibitionModes exhibitionMode = exhibition.getMode();
+            if (exhibitionMode.equals(ExhibitionModes.ACTIVE)) {
+                model.addAttribute("modeValue", "This Exhibition is Active.Use this Link for non-active Exhibition");
+                return "/exhibition/maintenance";
```

Comment:
I'm confused. Why does this go to the maintenance site if the exhibition is active?